docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,511 @@
+    > **Extension SDK** should be updated to the latest version.
+
+
+## Flash Tizen Images with IoT Setup Manager
+
+IoT Setup Manager helps you to easily install Tizen on your IoT device using your computer. It helps to get your device running and connected to the local network. After your device is connected to the local network, you can start developing and testing the Tizen IoT applications.
+
+
+
+### Launch IoT Setup Manager
+
+You can launch the IoT Setup Manager with or without using Tizen Studio:
+
+#### Launch using Tizen Studio
+
+From Tizen Studio, you can launch the IoT Setup Manager in the following two ways:
+
+-   On the menu bar, go to **IoT > IoT Setup Manager**.
+-   On the toolbar, click the IoT Setup Manager icon as shown in the following image:
+
+    ![IoT Setup Manager menu path](media/tizen_studio_ism.png)
+
+#### Launch without using Tizen Studio
+
+During installation, the Package Manager creates shortcuts for IoT Setup Manager, which are used to launch the IoT Setup Manager as a standalone application. To launch the IoT Setup Manager for:
+
+-   Windows, go to **Start Menu > Programs > Tizen Studio > Tools > IoT Setup Manager**.
+-   Ubuntu, go to **Dash (equivalent to Start Menu) > Applications > IoT Setup Manager**.
+
+    ![IoT Setup Manager Main](media/setup_manager_main_window.png)
+
+### Flash SD Card using IoT Setup Manager
+
+To flash Tizen on your SD card using the IoT Setup Manager:
+
+Profiles are an easy way to store image paths and network configuration information. After you save a profile, you can use it to flash your SD Card instead of specifying all information again.
+
+1.  Create a profile:
+    1.  Click **Create** to create a profile. The **Edit Profile** window appears.
+
+        ![IoT Setup Manager Edit](media/ism_edit_rpi4_headed.png)
+
+    2.  Provide the required information such as target, local paths to boot and platform image, and so on.
+
+       -   **Profile name**: Enter a profile name, for example, Rpi4_Headed_Profile
+       -   **Target**: Select the board and image type from the drop-down list. For example : RPI4_HEADED
+       -   **Boot image**: Browse for the Tizen Boot image available on your hard drive. You must select proper images depending on the board. [Downlaod Guide](#download-binaries)
+       -   **platform image**: Browse for the Tizen Platform image available on your hard drive. You must select proper images depending on the board. [Downlaod Guide](#download-binaries)
+       -   **RPM bundles**: 
+            - Add the bundle directory to the Bundles panel.
+            - Use this to install wifi-plugin. Download using [Installing Drivers](#installing-drivers). Unzip and add path to 32 bit rpm or 64 bit rpm according to image to install.
+            - Leave blank if no rpm installation required.
+       -   **WiFi**: Enter id and password to connect to wifi. (Wifi-plugin installation is required). If wifi or ethernet is not required, you can clear the check box.
```

Comment:
Suggestion:

**WiFi**: Enter the SSID and password to connect to Wi-Fi. (Wifi-plugin installation is required). If Wi-Fi or ethernet is not required, you can clear the respective check box.